google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -43,7 +43,7 @@ appengine.deployment.error.creating.staging.directory=There was an unexpected er
 appengine.generate.config.button=Generate...
 appengine.appyaml.location.label=app.yaml path\:
 appengine.dockerfile.location.browse.button=Browse for Your Dockerfile Location
-appengine.deployment.error.with.code=Deployment failed with exit code: {0}
+appengine.deployment.error.with.code=Deployment failed with exit code: {0}\nPlease make sure that you are using the latest version of the Google Cloud SDK. (See: https://cloud.google.com/sdk/)
```

Comment:
Since I have windows on the brain from the other PR I was just working - do we want to be safe here and use a platform independent line break (since windows typically does a crlf)?